workshops/templates/workshops/admin_dashboard.html

```diff
@@ -39,6 +40,8 @@
     <table class="table table-striped">
     {% for event in current_events %}
     <tr>
+      <td class="text-center" width="25px">{% if event.is_swc %}{% bootstrap_tag "SWC" %}{% endif %}</td>
+      <td class="text-center" width="25px">{% if event.is_dc %}{% bootstrap_tag "DC" %}{% endif %}</td>
```

Comment:
1. You forgot about "LC".
2. That's a lot of space.

And you had to add an annotation. Maybe we can fix this by extending `Tag` model? E.g. add `Tag.objects.carpentries()` that's equivalent to `Tag.objects.filter(name__in=['SWC', 'DC', 'LC'])`, and later we could use `{% for tag in event.tags.carpentries %}{% bootstrap_tag tag %}{% endfor %}`, although this may generate quite a few queries. Anyway, my point is that there might be a smarter way.